.gitignore

```diff
@@ -0,0 +1,3 @@
+#node_modules
```

Comment:
- [ ] Kindly correct the path to the `node_modules` folder, we could use as follow 👉🏽 `/node_modules` (**without a dot** at the beginning)